pom.xml

```diff
@@ -375,17 +375,33 @@
         </executions>
       </plugin>
 
-      <!-- Find Bugs -->
+      <!-- FindBugs -->
       <plugin>
         <groupId>org.codehaus.mojo</groupId>
         <artifactId>findbugs-maven-plugin</artifactId>
-        <version>2.5.2</version>
+        <version>3.0.2</version>
         <configuration>
           <excludeFilterFile>${findbugs.path}findbugs-exclude.xml</excludeFilterFile>
+          <includeFilterFile>${findbugs.path}findbugs-include.xml</includeFilterFile>
```

Comment:
I'm not sure why they were needed it in the first place, but it seems to work fine without them so they are now removed.